### PR TITLE
Add a <kbd> button to the editor toolbar

### PR DESCRIPTION
**Pull request by a2sheppy**
This adds an option to the toolbar, adjacent to the `<code>` button,
which applies the `<kbd>` element to content. The `<pre>` button is
also moved to the right end of the toolbar button group it's in,
to place it closer to the syntax highlighting buttons.

This resolves these two bugs:

* https://bugzilla.mozilla.org/show_bug.cgi?id=1209767
* https://bugzilla.mozilla.org/show_bug.cgi?id=1251506

### CODE REVIEW

**Comment by schalkneethling** [comment]
After rebuilding the CKEditor source I can confirm that the editor toolbar is updated as expected:

screenshot 2019-01-07 at 15 04 39

@a2sheppy Just want to confirm that all relevant files are in the PR. The docs at https://kuma.readthedocs.io/en/latest/ckeditor.html#committing-changes state the following:

```
When updating CKEditor, adding a plugin, or changing the configuration, CKEditor should be rebuilt, and the results added to a new pull request.
```

Are there additional build files that need to be added to this PR, or is it not applicable? Thanks!

**Comment by a2sheppy** [comment]
Updated with the built CKEditor.

File: kuma/static/js/libs/ckeditor/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "git.ignoreLimitWarning": true
+}
```
Comment (schalkneethling) [inline kuma/static/js/libs/ckeditor/.vscode/settings.json]: @a2sheppy Was this intentionally added?

**Comment by schalkneethling** [comment]
Seems like a lot more files was added causing a whole list of conflicts. I wonder if all of these are required? https://kuma.readthedocs.io/en/latest/ckeditor.html#committing-changes

@jwhitlock Thoughts?

**Comment by a2sheppy** [comment]
So, there's been another change to CKEditor since I initially submitted this PR, apparently. I will attempt to deal with it.

**Comment by a2sheppy** [comment]
I have sadly exceeded my git skills at this point and am not sure how to proceed to fix this. I'm sorry about that. :(

**Comment by jwhitlock** [comment]
I don't think it would be a good use of your time to try to repair this PR. 

Here's the quick version:

1. Start over with a branch from the current master on the Kuma repo
2. Cherry-pick a941e9a6c30a4222d069b618e1843a4b31a4a2da onto that branch
3. Re-build
4. Submit a new PR

**Comment by a2sheppy** [comment]
Hoping someone will be able to help me clean this up.

**Comment by a2sheppy** [comment]
Will do. That's what I was thinking.

**Pull request closed without merging**